MoMMI/Modules/github.py

```diff
@@ -818,25 +777,20 @@ def gh_register_help() -> None:
         if not is_repo_valid_for_command(repo_config, channel, prefix):
             continue
 
-        issues = []
-        i = 1
-        while 1:
-            url = github_url(f"/repos/{repo}/issues")
-            part_issues = await get_github_object(url, {"labels" : labels, "page" : i, "per_page" : "100"})
-            if part_issues:
-                issues += part_issues
-                i += 1
-            else:
-                break
+        url = github_url(f"/repos/{repo}/issues")
 
-        if emote:
-            sort = sorted(issues, key=lambda i: get_github_object(f"{i.url}/reactions",{"content" : emote, "per_page" : "100"}).len)[ranking_limit:]
-        else:
-            sort = issues
+        page_get = await session.get(url, {"labels" : labels, "per_page" : "100"})
+        maxpage = re.search(page_get.headers["Link"], REG_GIT_HEADER_PAGENUM).group(1)
+        pagenum = random.randrange(1, maxpage)
+
+        issue_page = await get_github_object(url, {"labels" : labels, "page" : pagenum, "per_page" : "100"})
+        rand_issue = await random.choice(issue_page).number
```

Comment:
You can't use `.` to access dictionary elements in Python.

Reply:
fixed